### PR TITLE
add config for web init and setServerURL for web

### PR DESCRIPTION
**Pull request by jaredmixpanel**
This PR addresses: https://github.com/mixpanel/mixpanel-flutter/issues/70

Also: https://github.com/mixpanel/mixpanel-flutter/issues/67

### CODE REVIEW

File: lib/mixpanel_flutter_web.dart [hunk 1]
```diff
@@ -134,7 +142,24 @@ class MixpanelFlutterPlugin {
...
+
+  Object handleGetConfig(MethodCall call) {
+    return get_config();
+  }
+
+  void handleSetServerURL(MethodCall call) {
+    Map<Object?, Object?> args = call.arguments as Map<Object?, Object?>;
+    String serverURL = args['serverURL'] as String;
+    set_config(js.jsify({'api_host': serverURL}));
   }
```
Comment (jaredmixpanel) [inline lib/mixpanel_flutter_web.dart:163]: unsure if this is appropriate or not? maybe just adding the config interfaces is sufficient?

**Review by matthoare013 — COMMENTED** [review]
generally looks good. I think you need to bump the version number pubspec.yaml

**Comment by hamirshekhawat** [comment]
Thanks for this change. It is gonna be a life saver!

**Review by zihejia — APPROVED** [review]
LGTM

**Pull request merged**